community/modules/scheduler/schedmd-slurm-gcp-v5-controller/variables.tf

```diff
@@ -550,6 +550,19 @@ variable "instance_image" {
   }
 }
 
+variable "instance_image_override" {
```

Comment:
1. Let's discuss offline "override" as the terminology to use. I like having the variable being lexically close to "instance_image."
2. I think the text should steer the user toward a statement like "the image must be built to be compatible with release according to LINK-TO-URL" rather than "custom image that is not supported".

Reply:
Same as above.